corehq/apps/reports/static/reports/js/filters/main.js

```diff
@@ -189,6 +191,34 @@ hqDefine("reports/js/filters/main", [
                 });
             }
         });
+        $('.report-filter-form-drilldown').each(function(i, el) {
+            var $el = $(el), data = $el.data();
+            if (!data.isEmpty) {
+                var model = drilldownOptions.DrilldownOptionFilterControl({
+                    drilldown_map: data.optionMap,
+                    controls: data.controls,
+                    selected: data.selected,
+                    notifications: data.notifications,
+                });
+                $el.koApplyBindings(model);
+                model.init();
+            }
+
+            if (data.unknownAvailable || data.displayAppType) {
+                advancedFormsOptions.advanceFormsOptions(
```

Comment:
Nitpick and you didn't introduce this, but for clarity can you rename this to "advancedFormsOptions" with a d?

Reply:
heh, already did generally, but i guess i missed this spot